From af3dcaefbc893e8f74b1f0bd410e38044e6cfa88 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 02 Oct 2018 20:59:10 +0300
Subject: [PATCH] hl-fw-loader: add pciutils dependency

hl-fw-loader should gain the standalone functionality for working
with goya w/o KMD present.
As one of the first steps add pciutils library for accessing the
PCI buses and devices.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/dl/pciutils-3.5.5.tar.xz b/dl/pciutils-3.5.5.tar.xz
new file mode 100644
index 0000000..951c760
--- /dev/null
+++ b/dl/pciutils-3.5.5.tar.xz
Binary files differ
diff --git a/package/hl-fw-loader/hl-fw-loader.mk b/package/hl-fw-loader/hl-fw-loader.mk
index cd21c0b..6c565d8 100644
--- a/package/hl-fw-loader/hl-fw-loader.mk
+++ b/package/hl-fw-loader/hl-fw-loader.mk
@@ -1,7 +1,7 @@
 HL_FW_LOADER_SITE = hl-fw-loader
 HL_FW_LOADER_LICENSE = unclear
 
-HOST_HL_FW_LOADER_DEPENDENCIES = specs spdlog host-libopenssl
+HOST_HL_FW_LOADER_DEPENDENCIES = specs spdlog host-libopenssl host-pciutils
 
 HOST_HL_FW_LOADER_CONF_OPTS = -DTESTS_ENABLED=OFF -DCMAKE_EXE_LINKER_FLAGS="-static" -DCMAKE_BUILD_TYPE="Release"
 
